JS/modal.js

```diff
@@ -0,0 +1,113 @@
+    featured_image: "./desktop-img-1.png",
+    lang_list: ["javascript", "html", "css"],
+    breadcrumbs: ["FACEBOOK", "Full Stack Dev", "2015"],
+    source:
+      "https://github.com/Bantealem/Portfolio-setup-and-mobile-version-skeleton",
+    live_demo:
+      "https://bantealem.github.io/Portfolio-setup-and-mobile-version-skeleton/",
+  },
+  {
+    title: "Uber Navigation",
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required.",
+    details:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    featured_image: "./desktop-img2.png",
+    lang_list: ["javascript", "html", "css"],
+    breadcrumbs: ["Uber", "Lead Developer", "2018"],
+    source:
+      "https://github.com/Bantealem/Portfolio-setup-and-mobile-version-skeleton",
+    live_demo:
+      "https://bantealem.github.io/Portfolio-setup-and-mobile-version-skeleton/",
+  },
+];
+
+const breadcrumbsHtml = (workObj, key, bread = "") => {
+  /* eslint-disable no-restricted-syntax */
+  for (const breadcrumb in workObj[key].breadcrumbs) {
+    if (breadcrumb) {
+      bread += `<li>${workObj[key].breadcrumbs[breadcrumb]}</li>`;
+    }
+  }
+  /* eslint-enable no-restricted-syntax */
+  return bread;
+};
+
+const languageHtml = (workObj, key, langList = "") => {
+  /* eslint-disable no-restricted-syntax */
+  for (const lang in workObj[key].lang_list) {
+    if (lang) {
+      langList += `<li><span>${workObj[key].lang_list[lang]}</span></li>`;
+    }
+  }
+  /* eslint-enable no-restricted-syntax */
+  return langList;
+};
+
+// let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+// if I want the first value which is 1, I can use arr[0]
```

Comment:
- [ ] I have realized that in your modal.js you have a lot of commented-out code as inline comments. Kindly note that according to the javascript [best practices for the browser](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) number 7, You are supposed to remove all commented and unwanted code from your code base as it makes it cluttered and messy. Strive to always make your code as simple as possible so that you do not need to explicitly leave inline comments.

**Kindly make sure that you remove all commented out code as inline comments**